root/lib/systemd/system/titus-sidecar-metatron-sync@.service

```diff
@@ -10,7 +10,7 @@ PartOf=titus-container@%i.target
 [Service]
 EnvironmentFile=/var/lib/titus-environments/%i.env
 # Run as root (UID 0, GID 0) and with CAP_DAC_OVERRIDE so that containers with a `USER` instruction work
-ExecStart=/usr/bin/runc --root /var/run/docker/runtime-${TITUS_OCI_RUNTIME}/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE ${TITUS_CONTAINER_ID} /titus/metatron/bin/titus-metatrond
+ExecStart=/usr/bin/runc --root /var/run/docker/runtime-${TITUS_OCI_RUNTIME}/moby exec --user 0:0 --cap CAP_DAC_OVERRIDE ${TITUS_CONTAINER_ID} /sbin/docker-init -s -- /titus/metatron/bin/titus-metatrond
```

Comment:
Can you add env variable overrides for stderr/stdout redirect off (and make tini to respect `""`) so that the output goes to the journal before?

Also can you add a HANDOFF=true always here, because there is no reason to keep tini in the process tree for this case

Reply:
(Oh you said that in the discription working on interim, carry on)

Reply:
Changed